core/examples/json/ProcessList.json

```diff
@@ -1,17 +1,37 @@
-[
+{
+  "processList": [
```

Comment:
Since the API request is `GET /processes`, shouldn't the name be `"processes"` instead of `"processList"` ?

Reply:
Yeah ... probably the same change for jobs too.  I'll update the PR now ... before @bpross-52n tried to merge.

Reply:
Done!  Changed "processList" -> "processes" and "jobList" -> "jobs".